### DISCUSSION TITLE
Did dotfiles quit working today?

### DISCUSSION THREAD

**Discussion by bartr** [community/community#3507] [Codespaces] [answered]
Is it just me or did dotfiles quit working today?

**Comment by bartr** [comment, marked as answer]
It looks like they moved from ~/dotfiles to /workspaces/.codespaces/.persistedshare/dotfiles

the repo gets cloned but install.sh doesn't run - any suggestions?

**Reply by srivatsn** [reply]
@edgonmsft @jkeech any thoughts?

**Reply by jkeech** [reply]
@bartr, thanks for reporting! We're merging Codespaces container orchestration logic with VS Code Remote - Containers to provide more consistency between both. This is a bug in that new implementation, which has been fixed and will be rolled out tomorrow. Note that this new implementation is not available to all Codespaces users yet, so most users won't run into this problem -- only internal Microsoft users would hit this issue for the next day until the fix rolls out.
